Northwind/Northwind/Controllers/ProductController.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.EntityFrameworkCore;
+using Northwind.Models.ViewModels;
+using NorthWind.Data;
+using NorthWind.Models;
+using NorthWind.Models.ViewModels;
+
+namespace NorthWind.Controllers
+{
+    public class ProductController : Controller
+    {
+        private readonly ApplicationDbContext _context;
+        private readonly IConfiguration _configuration;
+
+        public ProductController(ApplicationDbContext context, IConfiguration configuration)
+        {
+            _context = context;
+            _configuration = configuration;
+        }
+
+        public async Task<IActionResult> Index()
+        {
+            var pageSize = _configuration.GetValue<int>("PAGE_SIZE");
```

Comment:
If you're interested give it a try.

https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-8.0